### PR TITLE
Canwaf/issue263 return type binding now for as_pandas() and as_databaker()

### PR DESCRIPTION
**Pull request by canwaf**
This will help with python linting, type hints, and using pylance or pywrite.

### CODE REVIEW

File: gssutils/transform/download.py [hunk 1]
```diff
@@ -43,10 +43,10 @@ def open(self):
         stream.decode_content = True
         return stream
 
-    def as_databaker(self, **kwargs):
+    def as_databaker(self, **kwargs) -> Union[List[xypath.Table], xypath.Table]:
```
Comment (robons) [inline gssutils/transform/download.py:46]: Does `_get_simple_databaker_tabs` ever return a single `xypath.Table` outside of a list?

Comment (canwaf) [reply gssutils/transform/download.py:46]: Without delving any deeper, I didn't want to write it so that it couldn't. I believe it returns a list of one.

Comment (robons) [reply gssutils/transform/download.py:46]: I believe it's only returning lists, so I'd leave it as `List[xypath.Table]` without the union to remove ambiguity.

Both of the branches in `_get_simple_databaker_tabs` force iterables -> lists before returning their values, so it's safe to specify that it's just returning a list.

Comment (canwaf) [reply gssutils/transform/download.py]: Union removed.

**Review by robons — APPROVED** [review]

**Pull request merged**